src/gcstats/gc_stats.c

```diff
@@ -80,7 +80,7 @@ static int xdebug_gc_collect_cycles(void)
 #else
 	run->collected = GC_G(collected) - collected;
 #endif
-	run->duration = xdebug_get_utime() - start;
+	run->duration = (xdebug_get_nanotime() - start) / (double)NANOS_IN_SEC;
```

Comment:
There is no reason this should still be stored as a double in seconds. We can just do the conversion at output instead in line 185, where a division is applied already anyway.

Reply:
fixed